pkg/policy/policy.go

```diff
@@ -99,3 +100,8 @@ func (s *SearchContext) String() string {
 func (s *SearchContext) CallDepth() string {
 	return strconv.Itoa(s.Depth * 2)
 }
+
+// Translator is an interface for altering policy rules
```

Comment:
User input, e.g. policy rules, should never be modified.

Reply:
Same reply as above, should the comment here be worded in a way that should ensure that this shouldn't be used on user input?